papers/numba/array_vs_list/array_vs_list.rst

```diff
@@ -0,0 +1,160 @@
+Arrays vs. Lists
+----------------
+
+Lists are a very common data structure in Python, providing a way to represent
+an ordered sequence of values.  They are incredibly convenient, allowing
+elements to be read and replaced quickly.  Elements can be inserted into and
+appended to lists as well, though much more slowly.  In fact, the convience of
```

Comment:
Reading this I was wondering if this related to Python performance or Numba performance. In plain Python, the slowness of appending to lists isn't really noticeable (it's an amortized O(1) operation, and the overhead of memory management isn't really a problem since everything is interpreted anyway).